src/antsibull_core/ansible_core.py

```diff
@@ -80,14 +85,21 @@ async def get_release_info(self) -> dict[str, t.Any]:
         """
         # Retrieve the ansible-base and ansible-core package info from pypi
         tasks = []
-        for package_name in ("ansible-core", "ansible-base"):
-            query_url = urljoin(self.pypi_server_url, f"pypi/{package_name}/json")
+        for a_package_name in ("ansible-core", "ansible-base"):
+            if package_name is not None and a_package_name != package_name:
+                continue
```

Comment:
This seems unnecessary. Why not just loop over `(package_name,)` when `package_name` is not None?

Reply:
c8e199c0e08d7b2e8d1153428545330d12c0d642.